albx/session03/list_lab.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+#Learn basic Python list io
+#http://uwpce-pythoncert.github.io/IntroToPython/exercises/list_lab.html#exercise-list-lab
+
+
+fruit = [ "Apples", "Pears", "Oranges", "Peaches" ]
+print("The current list contains",len(fruit),"fruit:", fruit)
+
+response = input("Which fruit would you like to append to the list? ")
+fruit.append(response)
+print("The current list contains",len(fruit),"fruit:", fruit)
+
+response = int(input("Please select a particular fruit by providing a number: "))
+print("The", response, "fruit is ", fruit[response-1])
+
+fruit = ["Mandarins"] + fruit
+print("The current list contains",len(fruit),"fruit:", fruit)
+
+fruit.insert(1, "Pineapples")
+print("The current list contains",len(fruit),"fruit:", fruit)
+
+print("Fruits in the list that begin with 'P': ")
+for i in range(len(fruit)):
+    if fruit[i][0] == "P": print(fruit[i])
+
+print("The current list contains",len(fruit),"fruit:", fruit)
+fruit.remove(fruit[-1])
+print("The current list contains",len(fruit),"fruit:", fruit)
+
+response = input("Which fruit would you like to delete from the list? ")
+if response in fruit:
+        fruit.remove(response)
+print("The current list contains",len(fruit),"fruit:", fruit)
+
+print("The current list contains",len(fruit),"fruit:", fruit)
+fruit = fruit * 2
+print("The current list contains",len(fruit),"fruit:", fruit)
+
+response = []
+while (response not in fruit):
+    try:
+        response = input("Which fruit would you like to delete from the list? ")
+    except ValueError:
+        print("Fruit not in list")
+        continue
+    else:
+        break
```

Comment:
I don't think this else/break would ever get called.  You could safely remove:
```
    continue
else:
    break
```
The `continue` command will bring you to the next iteration of the loops, but as you have no code after your `print("Fruit not in list")`, its going to do that anyway.